app/views/sections/items/ItemPackagingSealChoiceView.scala.html

```diff
@@ -48,8 +48,10 @@
         @messages("itemPackagingSealChoice.p1")
       }
 
-      @p() {
-        @Html(messages("itemPackagingSealChoice.p2", strong(Html(packagingDescription), withFullStop = true)))
+      @if(!isBulk) {
+        @p() {
+            @Html(messages("itemPackagingSealChoice.p2", packagingIndex.displayIndex, itemIndex.displayIndex) + strong(Html(packagingDescription + ":")) + strong(Html(packagingQuantity), withFullStop = true))
```

Comment:
String interpolation would be good here:
```suggestion
            @Html(messages("itemPackagingSealChoice.p2", packagingIndex.displayIndex, itemIndex.displayIndex) + strong(Html(s"$packagingDescription: $packagingQuantity")), withFullStop = true))
```